README.md

```diff
@@ -35,16 +37,74 @@ Send an email with `send-email`:
 
 It takes the time of a POST request.
 
-TODO:
-
-- [X] make the "to" parameter accept a list of addresses.
-- [X] add "reply_to". It is a cons cell / a list with: an email address, a name.
-
+# Updated usage
+## API KEY
+After obtaining an API key from [SendGrid](https://sendgrid.com/), you can either 
+### Provide the API key via an Operating System environment variable
+```lisp
+CL-USER> sendgrid:*api-key-environment-variable-name*
+"SENDGRID_API_KEY" ; default environment variable name, you can change it.
+```
+Then you can do it by OS built-in facility, such as
+```bash
+export SENDGRID_API_KEY=your_sendgrid_api_key_value
+```
+Or set it using UIOP
+```lisp
+CL-USER> (setf (uiop:getenv sendgrid:*api-key-environment-variable-name*) your-api-key-value)
+```
+### Provide the API key when calling ```sendgrid:send-mail```
```

Comment:
nitpick (here and below): online code only needs one backquote: `code`, not 3.

Reply:
thanks! updated in [Update README.md](https://github.com/vindarel/cl-sendgrid/pull/3/commits/69dabf46943d055ced2337eb9c1152f3e09bcaca)